src/clr-angular/utils/drag-and-drop/droppable/droppable.ts

```diff
@@ -34,7 +34,7 @@ export class ClrDroppable<T> implements OnInit, OnDestroy {
     this.droppableEl = this.el.nativeElement;
   }
 
-  private isDraggableMatch: boolean = false;
+  isDraggableMatch: boolean = false;
```

Comment:
Do host bindings have to be public with the compiler check?

Reply:
Yes, the app under `src/dev` (and any other application with `fullTemplateTypeCheck`) does not compile with private bindings. So if you follow the instructions in the docs and use `clrDroppable` your app won't compile. See issue #3932